src/test/Components/CreateImageWizardV2/CreateImageWizard.compliance.test.tsx

```diff
@@ -0,0 +1,209 @@
+    //   await clickNext();
+    //  await screen.findByRole('heading', {
+    //    name: /Additional Red Hat packages/i,
+    //   });
+    //  await screen.findByText(/no packages added/i);
+  });
+
+  test('create an image with an oscap profile', async () => {
+    await setup();
+
+    // select aws as upload destination
+    await user.click(await screen.findByTestId('upload-aws'));
+    await clickNext();
+
+    // aws step
+    await user.click(
+      await screen.findByRole('radio', {
+        name: /manually enter an account id\./i,
+      })
+    );
+
+    await user.type(
+      await screen.findByRole('textbox', {
+        name: 'aws account id',
+      }),
+      '012345678901'
+    );
+
+    await clickNext();
+    // skip registration
+    await user.click(await screen.findByLabelText('Register later'));
+    await clickNext();
+
+    // Now we should be at the OpenSCAP step
+    await screen.findByRole('heading', { name: /OpenSCAP/i });
+
+    await user.click(
+      await screen.findByRole('textbox', {
+        name: /select a profile/i,
+      })
+    );
+
+    await user.click(
+      await screen.findByText(
+        /cis red hat enterprise linux 8 benchmark for level 1 - workstation/i
+      )
+    );
+    // check that the FSC contains a /tmp partition
+    await clickNext();
+    //    await screen.findByRole('heading', { name: /File system configuration/i });
+    //   await screen.findByText(/tmp/i);
+
+    // check that the Packages contain a nftable package
+    //  await clickNext();
+
+    //   await screen.findByRole('heading', {
+    //     name: /Additional Red Hat packages/i,
+    //   });
+    //   await screen.findByText(/nftables/i);
+    //    await screen.findByText(/libselinux/i);
+  });
+});
+
+describe('On Recreate', () => {
+  const setup = async () => {
+    renderWithReduxRouter('imagewizard/1679d95b-8f1d-4982-8c53-8c2afa4ab04c');
+  };
+  test('with oscap profile', async () => {
+    const user = userEvent.setup();
+    await setup();
+    await screen.findByRole('button', {
+      name: /review/i,
+    });
+    const createImageButton = await screen.findByRole('button', {
+      name: /create image/i,
+    });
+    await waitFor(() => expect(createImageButton).toBeEnabled());
+
+    // check that the FSC contains a /tmp partition
+    // There are two buttons with the same name but cannot easily select the DDF rendered sidenav.
+    // The sidenav will be the first node found out of all buttons.
+    const buttonsFSC = await screen.findAllByRole('button', {
+      name: /file system configuration/i,
+    });
+    await user.click(buttonsFSC[0]);
+    await screen.findByRole('heading', { name: /file system configuration/i });
+    await screen.findByText('/tmp');
+
+    // check that the Packages contain a nftable package
+    await clickNext();
+    await screen.findByRole('heading', {
+      name: /Additional Red Hat packages/i,
+    });
+    await screen.findByText('nftables');
+  });
+});
```

Comment:
This entire test shouldn't pass. Let's comment it out for now and add a comment to check if the right Wizard is tested when we implement the functionality in V2